backend/tracim_backend/applications/upload_permissions/schema.py

```diff
@@ -0,0 +1,185 @@
+import cgi
+import typing
+
+import marshmallow
+from marshmallow import post_load
+from marshmallow.validate import OneOf
+
+from tracim_backend.app_models.validator import bool_as_int_validator
+from tracim_backend.app_models.validator import strictly_positive_int_validator
+from tracim_backend.applications.upload_permissions.models import UploadPermissionType
+from tracim_backend.applications.upload_permissions.validators import (
+    upload_permission_email_validator,
+)
+from tracim_backend.applications.upload_permissions.validators import (
+    upload_permission_password_validator,
+)
+from tracim_backend.lib.utils.utils import DATETIME_FORMAT
+from tracim_backend.views.core_api.schemas import StrippedString
+from tracim_backend.views.core_api.schemas import UserDigestSchema
+from tracim_backend.views.core_api.schemas import WorkspaceIdPathSchema
+
+
+class UploadFiles(object):
+    def __init__(self, **files: cgi.FieldStorage) -> None:
+        self.files = files.values()
```

Comment:
Do you know type of `self.files` ? It can be interesting to add it

Reply:
This is typed, but that's true that's not so obvious that files.values are values of dict files which contain cgi.FieldStorage as typed before.

Reply:
Technically it is not type, it is inference type. So, as developper i have doubt about `self.files` type when reading this code. If it is possible to type it to help reader, do it ;)

Reply:
added typing.